src/Storages/StorageFile.cpp

```diff
@@ -279,22 +276,6 @@ std::unique_ptr<ReadBuffer> selectReadBuffer(
 
         ProfileEvents::increment(ProfileEvents::CreatedReadBufferOrdinary);
     }
-    else if (read_method == LocalFSReadMethod::io_uring && !use_table_fd)
```

Comment:
Why `storage_file_read_method=io_uring` supported had been reverted?

Reply:
because I wanted to quickly test the fix